domain/policy.go

```diff
@@ -310,6 +313,16 @@ func (p *Policy) RemoveSensitiveValues() {
 	}
 }
 
+func (p *Policy) GetStepByName(name string) *Step {
```

Comment:
can you also add unit tests for this to help the test coverage?

Reply:
Done